### PR TITLE
[Breaking] Rename IndicatorSet to Indicator

### PR DESCRIPTION
**Pull request by dourouc05**
Follows https://github.com/jump-dev/MathOptInterface.jl/pull/1475 and https://github.com/jump-dev/MathOptInterface.jl/issues/1459. 

I created a new file for v0.7, I thought this would be the most convenient (especially as this change cannot be part of v0.6.x).

### CODE REVIEW

File: schemas/mof.0.7.schema.json [hunk 1]
```diff
@@ -0,0 +1,878 @@
+{
+    "$schema": "https://json-schema.org/schema#",
+    "$id": "https://jump.dev/MathOptFormat/schemas/mof.0.6.schema.json",
```
Comment (odow) [inline schemas/mof.0.7.schema.json:3]:
```suggestion
    "$id": "https://jump.dev/MathOptFormat/schemas/mof.0.7.schema.json",
```

**Review by odow — COMMENTED** [review]
So there are couple of other things to do:
 - [ ] Update the examples to 0.7
 - [ ] Update the README

I really should setup CI to run this script: 
https://github.com/jump-dev/MathOptFormat/blob/master/python/mof.py

**Comment by dourouc05** [comment]
I also added a small fix for the Python script, as it could not find out the correct encoding (Python 3.8.8 x64 on Windows).

**Review by odow — COMMENTED** [review]
Looks good. But let's wait for the outcome of the discussion on the monthly call.

**Pull request merged**